.env

```diff
@@ -1,2 +1,6 @@
 DATABASE_URL = "mongodb+srv://test123:test123@cluster0.gczn8.mongodb.net/portifolio?retryWrites=true&w=majority"
```

Comment:
I think for security purposes, your MongoDB connecting URL has to be in the .env file.